src/items/deleteItemDialog.component.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormHelperText,
+  Typography,
+  Link as MuiLink,
+} from '@mui/material';
+import React from 'react';
+import { useDeleteItem } from '../api/item';
+import { Item } from '../app.types';
+import { AxiosError } from 'axios';
+import { useSystem } from '../api/systems';
+import { Link } from 'react-router-dom';
+
+export interface DeleteItemDialogProps {
+  open: boolean;
+  onClose: () => void;
+  item: Item | undefined;
+  onChangeItem: (Item: Item | undefined) => void;
+}
+
+const DeleteItemDialog = (props: DeleteItemDialogProps) => {
```

Comment:
Could the warning message be on a separate line, it feels too long

![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/5063a872-7236-4903-a96a-30ee230b7266)

Also I feel the warning could stand out a bit more although not sure how, an `<Alert>` may be too much?

Reply:
It doesn't look good on two lines. Alert is too much and orange text doesn't like right

Reply:
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/0568e5b4-1f36-48c6-a1f7-29c69f9d7ca0)

Reply:
What about a warning icon?
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/c2dcb73c-ea41-4b38-b07b-e569488ac621)

Reply:
The message in the dialog is already warning the user. The extra icon would be useless in this case as it would be need for all of the other dialogs anyways

Reply:
I'll raise this in the next user feedback meeting. if they want anymore indication of a warning.

Reply:
> Right now if you resize it it does this ![image](https://private-user-images.githubusercontent.com/90245114/297805539-0568e5b4-1f36-48c6-a1f7-29c69f9d7ca0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2NTY3NTcsIm5iZiI6MTcwNTY1NjQ1NywicGF0aCI6Ii85MDI0NTExNC8yOTc4MDU1MzktMDU2OGU1YjQtMWYzNi00OGM2LWExZjctMjljNjlmOWQ3Y2EwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE5VDA5MjczN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3NzVjNWIxY2QxZDE5ZDNkNzIzZTMzZWMxMGZmZTcyYWI1NjZhN2ZiNjJkY2I0ZDczOGVhMDNkNjE0OWJiN2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jENuW9PzuR7BJMzL_hR_nVwG-MK2mfidbL8qi-6zMhk)

I'll fix this now

Reply:
I'll create an issue to add the warning icon to all of the delete dialogs as would be a useful indicator especially because it's currently a permanent delete

Reply:
Yeah sure, I was only suggesting it as a way to just glance and see there is something different about it, a warning indicator on all should hopefully make sure people read the message and don't just glance first and delete.

Reply:
#252